AwsCryptographicMaterialProviders/runtimes/java/build.gradle.kts

```diff
@@ -161,11 +164,11 @@ tasks.shadowJar {
     }
 
     configurations {
-        runtimeClasspath {
+        shadow {
             dependencies {
                 // We want to package this version of BC since it is the one the Primitives depends on.
                 // These dependencies need to remain in sync with one another.
-               include(dependency("org.bouncycastle:bcprov-jdk18on:1.72"))
+                dependency("org.bouncycastle:bcprov-jdk18on:1.72")
```

Comment:
Do we actually still need this? What is it actually doing if we are explicitly having this package depend on BC on L74?

Reply:
ooh good point; i guess something to think about is that instead of including the dependencies in the jar; we could add them like this but then that results in having to keep track of dependencies in too many places. i'll test and see if everything still works by removing this